changelog-entries/244.md

```diff
@@ -0,0 +1 @@
+- Removed the default `-j 4` option from the wmake command. Instead, documented the `WM_NCOMPPROCS` option of OpenFOAM. [#244](Remove -j 4 flag, document WM_NCOMPPROCS)
```

Comment:
```suggestion
- Removed the default `-j 4` option from the wmake command. Instead, documented the `WM_NCOMPPROCS` option of OpenFOAM. [#244](https://github.com/precice/openfoam-adapter/pull/244)
```

Reply:
Thanks for the fix! Please check again and approve, if ok.